apps/astarte_trigger_engine/lib/astarte_trigger_engine/events_consumer.ex

```diff
@@ -57,6 +57,7 @@ defmodule Astarte.TriggerEngine.EventsConsumer do
           timestamp
       end
 
+    :telemetry.execute([:astarte, :trigger_engine, :consumed_http_event], %{}, %{realm: realm})
```

Comment:
This should be `:consumed_event`. Right now we only support HTTP events but there will be more in the future, and their specific type is determined down the pipeline.